schedule/functional/toolchain.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: toolchain_zypper
+description: >
+    Maintainer: zluo
+    toolchain tests for sles and opensuse
+conditional_schedule:
+    bootloader:
+        BACKEND:
+            'svirt':
+                - installation/bootloader_zkvm
+    kdump_and_crash:
+        DISTRI:
+            sle:
+                - console/kdump_and_crash
```

Comment:
Not sure why SLE is different there. IMO, `kdump_and_crash` does not make much sense in a toolchain test.

Reply:
And I also checked for sle functional on OSD, kdump_and_crash just appears in job 'toolchain_zypper' atm.